views/index.html

```diff
@@ -9,85 +9,83 @@
   </head>
   <body>
 
-    <div class="calculator">
-      <div class="display">
-        <p id="numbers">0</p>
-    </div>
+    <div class="page-container">
+      <div class="calculator-body">
+      <div class="display"><p id="numbers">0</p></div>
+        <div id="calculator-numbers">
+          <div class="calclulator-row1">
+            <div class="calculator-button">
+              <button class="calculator-ac">AC</button>
+            </div>
+            <div class="calculator-button">
+              <button class="calculator-positive-negative">+/-</button>
+            </div>
+            <div class="calculator-button">
+              <button class="percent">%</button>
+            </div>
+            <div class="calculator-button">
+              <button class="calculator-orange-button white-operator divide">÷</button>
+            </div>
+          </div>
 
-  <div id="calculator-numbers">
+          <div class="calculator-row2">
+            <div class="calculator-button">
+              <button id="button7" onclick='eventLogger'>7</button>
```

Comment:
When you get to Step 2, come ask about Event Delegation.  You don't need an onClick on every button.  You can add an Event Listener to the parent (in your case the div with the id #calculator-numbers).  Then, you can check in Javascript for which child was clicked.